gravitee-apim-console-webui/src/management/api/proxy/components/health-check-form/api-proxy-health-check-form.component.spec.ts

```diff
@@ -0,0 +1,347 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { NoopAnimationsModule } from '@angular/platform-browser/animations';
+import { HttpTestingController } from '@angular/common/http/testing';
+import { HarnessLoader } from '@angular/cdk/testing';
+import { TestbedHarnessEnvironment } from '@angular/cdk/testing/testbed';
+import { InteractivityChecker } from '@angular/cdk/a11y';
+import { MatSlideToggleHarness } from '@angular/material/slide-toggle/testing';
+import { MatSelectHarness } from '@angular/material/select/testing';
+import { MatInputHarness } from '@angular/material/input/testing';
+import { GioFormHeadersHarness } from '@gravitee/ui-particles-angular';
+import { MatIconTestingModule } from '@angular/material/icon/testing';
+import { MatButtonHarness } from '@angular/material/button/testing';
+
+import { ApiProxyHealthCheckFormComponent } from './api-proxy-health-check-form.component';
+import { ApiProxyHealthCheckFormModule } from './api-proxy-health-check-form.module';
+
+import { CurrentUserService } from '../../../../../ajs-upgraded-providers';
+import { User } from '../../../../../entities/user';
+import { GioHttpTestingModule } from '../../../../../shared/testing';
+import { HealthCheck } from '../../../../../entities/health-check';
+
+describe('ApiProxyHealthCheckFormComponent', () => {
+  const currentUser = new User();
+  currentUser.userPermissions = ['api-definition-u'];
+
+  let fixture: ComponentFixture<ApiProxyHealthCheckFormComponent>;
+  let loader: HarnessLoader;
+  let component: ApiProxyHealthCheckFormComponent;
+  let httpTestingController: HttpTestingController;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports: [NoopAnimationsModule, GioHttpTestingModule, ApiProxyHealthCheckFormModule, MatIconTestingModule],
+      providers: [{ provide: CurrentUserService, useValue: { currentUser } }],
+    }).overrideProvider(InteractivityChecker, {
+      useValue: {
+        isFocusable: () => true, // This checks focus trap, set it to true to  avoid the warning
+      },
+    });
```

Comment:
No warning. You should be able to remove it
```suggestion
    });
```